src/classes/BGE_DataImportBatchEntry_CTRL.cls

```diff
@@ -186,17 +186,49 @@ public with sharing class BGE_DataImportBatchEntry_CTRL {
     * @return DataImportModel with updated data import record row and updated totals for the batch
     */
     @AuraEnabled
-    public static String runDryRun(Id dataImportId, Id batchId) {
+    public static String runDryRun(List<Id> dataImportIds, Id batchId) {
```

Comment:
Why does this one need to take a list now, if we're using a different method for mass? Couldn't this go back to a singleton?

Reply:
Are we going to keep runDryRun as-is needing a list? getDataImportQuery ultimately still requires a list